### PR TITLE
Entity Export destination Save Draft and Delete

### PR DESCRIPTION
**Pull request by Lizzzzi**
https://trello.com/c/nJsbMqUy

### CODE REVIEW

File: test/specs/EntityExportDestination.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        const datetime = $("#datetime");
+        string.setValue("Abc");
+        text.setValue("Qwerty");
+        int.setValue(123);
+        decimal.setValue(45.67);
+        date.click();
+        datetime.click();
+    });
+    it('07 Should save record as a draft',  () => {
+        const saveButt = $("#pa-entity-form-draft-btn");
```
Comment (Burdishka) [inline test/specs/EntityExportDestination.js:59]: saveButton or saveDraftButton?

File: test/specs/EntityExportDestination.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    });
+    it('10 Should open Action menu and delete record',  () => {
+        const menuButt = $("//tr[@data-index='0']/td/div/button");
+        menuButt.click();
+        browser.pause(2000);
+        const deleteButt = $("//a[contains(text(), 'delete')]");
+        deleteButt.click();
+    });
+    it('11 Should RecycleBin notification is displayed and equal 1',  () => {
+        //browser.pause(1000);
```
Comment (Burdishka) [inline test/specs/EntityExportDestination.js:83]: Commented lines of code in this case do not need

File: test/specs/EntityExportDestination.js [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+        int.setValue(123);
+        decimal.setValue(45.67);
+        date.click();
+        datetime.click();
+    });
+    it('07 Should save record as a draft',  () => {
+        const saveButt = $("#pa-entity-form-draft-btn");
+        saveButt.scrollIntoView();
+        saveButt.click();
+        browser.pause(2000);
```
Comment (Burdishka) [inline test/specs/EntityExportDestination.js:62]: try don’t use pauses, try use wait until

Comment (Lizzzzi) [reply test/specs/EntityExportDestination.js]: i fixed it

**Review by MariaMoroz — COMMENTED** [review]
use, please, Data-driven model and remove all pauses

**Review by Burdishka — APPROVED** [review]

**Pull request merged**